2025/executive-vote-2025-05-29-MKR-to-SKY-upgrade-phase-two.md

```diff
@@ -0,0 +1,440 @@
+|-----------|-------------:|-----------------------------------------------------------------------------------------------------------------------|
+| BLUE      | 330,000      | [0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf](https://etherscan.io/address/0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf) |
+| Cloaky    | 288,000      | [0x9244F47D70587Fa2329B89B6f503022b63Ad54A5](https://etherscan.io/address/0x9244F47D70587Fa2329B89B6f503022b63Ad54A5) |
+
+### Execute Spark Proxy Spell
+
+- **Authorization**: [Ecosystem Approval](https://forum.sky.money/t/may-29-2025-proposed-changes-to-spark-for-upcoming-spell/26372/3)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/may-29-2025-proposed-changes-to-spark-for-upcoming-spell/26372)
+
+If this executive proposal passes, then a Spark Proxy Spell will be executed at [$TBD](https://etherscan.io/address/TBD) with the following contents.
+
+#### Onboard Unichain to the Spark Liquidity Layer
+
+- **Authorization**: [Governance Poll 1253](https://vote.makerdao.com/polling/QmXjeJtw)
+- **Proposal**: [Forum Post](https://forum.sky.money/t/may-29-2025-proposed-changes-to-spark-for-upcoming-spell/26372)
+
+Onboard Unichain to the Spark Liquidity Layer with the following addresses and parameters.
+
+##### Addresses
+
+- L1_TOKEN_BRIDGE: [0xDF0535a4C96c9Cd8921d8FeC92A7680b281681d2](https://etherscan.io/address/0xDF0535a4C96c9Cd8921d8FeC92A7680b281681d2)
+- L2_TOKEN_BRIDGE: [0xa13152006D0216Fe4627a0D3B006087A6a55D752](https://uniscan.xyz/address/0xa13152006D0216Fe4627a0D3B006087A6a55D752)
+- L2_USDS: [0x7E10036Acc4B56d4dFCa3b77810356CE52313F9C](https://uniscan.xyz/address/0x7E10036Acc4B56d4dFCa3b77810356CE52313F9C)
+- L2_SUSDS: [0xA06b10Db9F390990364A3984C04FaDf1c13691b5](https://uniscan.xyz/address/0xA06b10Db9F390990364A3984C04FaDf1c13691b5)
+
+- L1_SSR_FORWARDER: [0x45d91340B3B7B96985A72b5c678F7D9e8D664b62](https://etherscan.io/address/0x45d91340B3B7B96985A72b5c678F7D9e8D664b62)
+- L2_SSR_RECEIVER: [0x4A71f81C6109230932978bAB7CA746f0be0C4580](https://uniscan.xyz/address/0x4A71f81C6109230932978bAB7CA746f0be0C4580)
+- L2_SSR_AUTH_ORACLE: [0x1566BFA55D95686a823751298533D42651183988](https://uniscan.xyz/address/0x1566BFA55D95686a823751298533D42651183988)
+
+- L2_GOV_RECEIVER: [0x7B8ee8b0fD62662F7FB1ac9e5E6cEAad5195A3bF](https://uniscan.xyz/address/0x7B8ee8b0fD62662F7FB1ac9e5E6cEAad5195A3bF)
+- L2_GOV_EXECUTOR: [0xb037C43b433964A2017cd689f535BEb6B0531473](https://uniscan.xyz/address/0xb037C43b433964A2017cd689f535BEb6B0531473)
+
+- L2_PSM: [0x7b42Ed932f26509465F7cE3FAF76FfCe1275312f](https://uniscan.xyz/address/0x7b42Ed932f26509465F7cE3FAF76FfCe1275312f)
+
+- L2_ALM_PROXY: [0x345E368fcCd62266B3f5F37C9a131FD1c39f5869](https://uniscan.xyz/address/0x345E368fcCd62266B3f5F37C9a131FD1c39f5869)
+- L2_ALM_CONTROLLER: [0x9B1BEB11CFE05117029a30eb799B6586125321FF](https://uniscan.xyz/address/0x9B1BEB11CFE05117029a30eb799B6586125321FF?tab=contract)
```

Comment:
```suggestion
- L2_ALM_CONTROLLER: [0x9B1BEB11CFE05117029a30eb799B6586125321FF](https://uniscan.xyz/address/0x9B1BEB11CFE05117029a30eb799B6586125321FF)
```

(If you want to go to the contract tab, seems like you need to add "?tab=contract#code")